src/controllers/createBlogController.js

```diff
@@ -0,0 +1,45 @@
+import mssql from "mssql";
+import { body, Result } from "express-validator";
+import jwt from "jsonwebtoken";
+
+const createBlogController = async (req, res) => {
+  try {
+    const authHeader = req.headers["authorization"];
+    const token = authHeader && authHeader.split(" ")[1];
```

Comment:
why we slit after empty string?

Reply:
const config = {
          headers: {
            Authorization: `Bearer ${token}`,
          },
        }; this is how i configured token in frontend and i used split to return the second element which is the token

Reply:
alright maybe split after 'Bearer'?